editor.planx.uk/src/@planx/components/Feedback/Public/Public.tsx

```diff
@@ -21,14 +26,39 @@ import { Feedback, FormProps } from "../model";
 import { StyledToggleButtonGroup } from "../styled";
 
 const FeedbackComponent = (props: PublicProps<Feedback>): FCReturn => {
+  const handleSubmitFeedback = async (
+    values: FormProps,
+    { resetForm }: FormikHelpers<FormProps>,
+  ) => {
+    const metadata = await getInternalFeedbackMetadata();
+    const feedback = {
+      userComment: values.feedback,
+      feedbackScore: values.feedbackScore,
+    };
+    const data = {
+      ...metadata,
+      ...feedback,
+      feedbackType: "component" as FeedbackView,
+    };
```

Comment:
```suggestion
    const metadata = await getInternalFeedbackMetadata();
    const data = {
      ...metadata,
      ...values,
    };
```

nit: We gain a small simplification and slightly lessen the mental load here by not having to convert `feedback` to `userComment`. I think we could use `userComment` as the field name to match the db column 👍 

Likewise, there's nothing stopping us adding the `feedbackType` field to the form and then not just presenting it to the user (not sure if this is actually a helpful step though!)

Reply:
Great suggestion! I think I would find having the feedbackType in the form more confusing to read in the code so have left that out though.